crates/wash-cli/src/bin/wash.rs

```diff
@@ -225,6 +233,14 @@ async fn main() {
         .with_env_filter(EnvFilter::from_default_env())
         .init();
 
+    // Implement clap_markdown for markdown generation of command line documentation
+    let mdargs = Cli::parse();
+
+    // Most straightforward way to invoke is probably `wash app list --markdown-help > help.md`
+    if mdargs.markdown_help {
+        clap_markdown::print_help_markdown::<Cli>();
+    };
```

Comment:
If someone has this set, can we assume this is all they want to do? As in, should we exit immediately after? 

Also, I wonder if we can mvoe that `Cli::parse();` *inside* the `if`?

Reply:
Added an exit in the most recent commit! My understanding is the parse needs to be outside the if, but I ended up removing the line (and moving the if statement down) to avoid double-parsing per Brooks' suggestion below.